installation/install.sh

```
@@ -2574,6 +2583,188 @@ install_morph_server(){
+    echo_subtitle "Prepare License Server db and user"
+    trap 'traplogger $LINENO "$BASH_COMMAND" $?'  DEBUG
+
+    local root_password="$MYSQL_NEW_ROOT_PASSWORD"
+    local user_password="$LICENSE_SERVER_DEFAULT_DB_PASSWORD"
+    local reset_user='n'
+
+    [ -n "$MYSQL_ROOT_PASSWORD" ] && root_password="$MYSQL_ROOT_PASSWORD"
+    [ -n "$LICENSE_SERVER_DB_PASSWORD" ] && user_password="$LICENSE_SERVER_DB_PASSWORD" && reset_user='y'
+
+    mysql -u root --password="$root_password" -e 'exit' >/dev/null 2>&1
+    [ $? -eq 0 ] || fail2 "\nCannot login mysql! If you have mysql root password, please add option '-P MYSQL_ROOT_PASSWORD'.\n"
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)